002_sdk_L464.py

```
@@ -0,0 +1,485 @@
+            for flow_msg in flow:
+                result = liqi.parse(flow_msg)
+                handler.parse(result)
+                tot += 1
+            history_msg = history_msg+flow
+            pickle.dump(history_msg, open(filename, 'wb'))
+        time.sleep(0.2)
+// {fact rule=code-injection@v1.0 defects=1}
+
+
+def replayWebSocket(filename='ws_dump.pkl'):
+    # 回放历史websocket报文，按顺序交由handler.parse
```

Ruff (0.13.3)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)